style.css

```diff
@@ -0,0 +1,107 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&family=Ubuntu:wght@300&display=swap");
+
+/**
+*! Color variables
+**/
+
+/**
+*! Removing default browser style
+**/
+* {
+  box-sizing: border-box;
+}
+
+* body {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+}
+
+* a {
+  text-decoration: none;
+}
+
+* li {
+  list-style: none;
+}
+
+/**
+*! Header styles inputs here
+**/
+.navbar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.logo {
+  font-weight: 700;
+  font-size: 1.125rem;
+  line-height: 1.25rem;
+  margin-left: 1rem;
+  color: #6070ff;
+}
+
+[class^=bar] {
+  display: block;
+  background-color: #6070ff;
+  width: 1.875rem;
+  height: 0.1875rem;
+  margin: 0.3125rem;
+  margin-left: 14rem;
```

Comment:
- [ ] If you decided on using the `justify-content: space-between;` then the only thing left to achieve the balance in right-left space around the navbar is by using `margin-right`  instead of the `margin-left`  to the hamburger menu.

Reply:
Done as  recommended